CHANGELOG.md

```diff
@@ -33,6 +33,21 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### Added
 
+- Support passsing the inital UTxO distribution as an Array and also get the KeyWallets as an Array when writing Plutip tests.
+To achieve this the following class instances were added ([#1018](https://github.com/Plutonomicon/cardano-transaction-lib/pull/1018)):
+  - `UtxoDistribution (Array InitialUTxOs) (Array KeyWallet)`
+  - `UtxoDistribution (Array InitialUTxOsWithStakeKey) (Array KeyWallet)`
+
+  Example usage:
```

Comment:
Sorry for bothering you again. A usage example would be fit in `doc/plutip-testing.md` (a link here is OK). The changelog is needed to help the users update quickly, so it must be succinct.

Reply:
Please remove the mentions of UtxoDistribution.

Reply:
there is already an usage example there, I've added the link and removed the mentions